std/experimental/logger/core.d

```diff
@@ -750,14 +748,42 @@ abstract class Logger
         string moduleName;
         /// the $(D LogLevel) associated with the log message
         LogLevel logLevel;
-        /// thread id of the log message
-        Tid threadId;
         /// the time the message was logged
         SysTime timestamp;
         /// the message of the log message
         string msg;
-        /// A refernce to the $(D Logger) used to create this $(D LogEntry)
+    }
+
+    /** LogEntry is an aggregation combining all information associated
+    with a log message. This aggregation will be passed to the method
+    writeLogMsg.
+    */
+    protected struct LogEntry
+    {
+        LogEntryBase base;
+        alias base this;
+
+        /// thread id of the log message
+        Tid threadId;
+        /// A reference to the $(D Logger) used to create this $(D LogEntry)
         Logger logger;
```

Comment:
This field was in original `LogEntry` and `LogEntry` was/is protected. Potentially this change can break user code. May be it is better to remove this field by separate commit, just in case.